block/internal/syncing/da_retriever.go

```diff
@@ -101,14 +101,14 @@ func (r *DARetriever) fetchBlobs(ctx context.Context, daHeight uint64) (coreda.R
 	defer cancel()
 
 	// Get namespaces
-	headerNamespace := []byte(r.config.DA.GetNamespace())
-	dataNamespace := []byte(r.config.DA.GetDataNamespace())
+	headerNamespace := r.config.DA.GetNamespace()
+	dataNamespace := r.config.DA.GetDataNamespace()
 
 	// Retrieve from both namespaces
 	headerRes := types.RetrieveWithHelpers(ctx, r.da, r.logger, daHeight, headerNamespace)
 
 	// If namespaces are the same, return header result
-	if string(headerNamespace) == string(dataNamespace) {
+	if headerNamespace == dataNamespace {
```

Comment:
Nit: Bytes.equal would be cleaner here.

Reply:
https://github.com/evstack/ev-node/pull/2683/commits/044cf0cf443229b25cb7a846ade911cf0b0aac7f